CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-alpha.16
+
+###### _Nov 14 2020_
+
+Big thanks to the 34 contributors who made this release possible. Here are some highlights ✨:
+
+- 📅 Migrate to the date picker in the lab (#22692) @dmtrKovalenko.
+  We have integrated the components inside the code infrastructure. In the coming days, we will migrate all the issues from [material-ui-pickers](https://github.com/mui-org/material-ui-pickers) and archive the repository. This migration will help provide first-class support for the date picker components. The component will stay in the lab as long as necessary to reach the high expectations bar we have for core components. You can find the [new documentation here](https://next.material-ui.com/components/pickers/).
+
+  While the sources are currently hosted in the [main repository](https://github.com/mui-org/material-ui), we might move them to the [x repository](https://github.com/mui-org/material-ui-x), depending on what is easier for the commercial date range picker. The date picker will stay open source no matter what.
+
+- 📚 Revamp the documentation for [the system](https://next.material-ui.com/system/basics/). The System contains CSS utilities for rapidly implementing custom designs. It now emphasizes the `sx` prop. It's ideal for adding one-off styles, e.g. padding. When pushed to its limits, it can be used to implement quickly a complete marketing page.
+- 👩‍🎨 Upgrade emotion to v11 (#23007) @mnajdova.
+- And many more 🐛 bug fixes and 📚 improvements.
+
+### `@material-ui/core@v5.0.0-alpha.16`
+
+#### Breaking changes
+
+- [TextField] Change default variant from standard to outlined (#23503) @mbrookes
+  Standard has been removed from the Material Design guidelines. [This codemod](https://github.com/mui-org/material-ui/tree/next/packages/material-ui-codemod#textfield-variant-prop) will automatically update your code.
+
+  ```diff
+  -<TextField value="Standard" />
+  -<TextField value="Outlined" variant="outlined" />
+  +<TextField value="Standard" variant="standard" />
+  +<TextField value="Outlined" />
+  ```
+
+- [Autocomplete] Remove `debug` in favor of `open` and dev tools (#23377) @eps1lon
+  There are a couple of simpler alternatives: `open={true}`, Chrome devtools ["Emulate focused"](https://twitter.com/sulco/status/1305841873945272321), or React devtools props.
+
+#### Changes
+
+- [Autocomplete] Use Popper with when disablePortal (#23263) @eps1lon
```

Comment:
```suggestion
- [Autocomplete] Use Popper when `disablePortal` (#23263) @eps1lon
```
I sometimes change what I want to say mid-sentence which leads to these confusing statements sometimes :smile: